docs/user/software/development/rocm.md

```diff
@@ -0,0 +1,189 @@
+---
+title: ROCm/HIP
+summary: A quick guide to getting set up for ROCm/HIP development on Solus
+---
+
+# ROCm/HIP
+
+ROCm is AMD's open-source software stack for GPU computation.
+
+Note that ROCm is not required in order for, say, your display or browser, to
+use GPU-acclerated rendering. These are more on the driver side of things and 
+are
+handled by the kernel and/or Mesa. ROCm is mainly focused on GPU-accelerated
+computing, such as GPU rendering in Blender or GPU-accelerated machine learning
+in PyTorch.
+
+## Install ROCm/HIP
+
+```bash
+sudo eopkg it rocm-hip rocm-opencl
+```
+
+If you are also developing with ROCm/HIP, install the
+development files and the `hipcc` compiler driver as well:
+```bash
+sudo eopkg it rocm-hip-devel
+```
+
+## Necessary Environment Variables
+
+It is recommended and safe to put these environment variables in your
+`~/.bashrc`:
+```bash
+export ROCM_PATH=/usr
+export HIP_PATH=/usr
+```
+
+If you're developing with ROCm/HIP, the following environment variables will
+save you a lot of hassle:
+```bash
+export HIP_DEVICE_LIB_PATH=/usr/lib64/amdgcn/bitcode
+export DEVICE_LIB_PATH=$HIP_DEVICE_LIB_PATH
+export HIP_PLATFORM=amd
+export HIP_RUNTIME=amd
+export HIP_ROCCLR_HOME=$ROCM_PATH
+```
+
+## Supported Hardware and GPU Architectures
+
+<!--
+ROCm is designed such that in order to for the compiled binaries to run on a 
+certain GPU model, during compiling one must compile with that GPU as the
+compilation target. 
+!-->
+
+ROCm is designed such that in order for a library to support N different GPU
```

Comment:
This is a [wall of text](https://developers.google.com/style/accessibility#ease-of-reading), nobody will read this. 

With that being said, is it even necessary to mention all of this? I mean, for end users what matters is whether the GPU architecture is supported or not: If it's supported, they'll continue their day without reading everything; if it's not supported, they'll reach out using the available support channels.

```suggestion

:::tip

Install and run `rocminfo`  to see your GPU architecture.

:::

The ROCm package included in Solus supports the following GPU architectures out of the box:

- `gfx803`
- `gfx900`
- `gfx906`
- `gfx908`
- `gfx90a`
- `gfx1010`. This architecture can run programs compiled for `gfx101*` GPUs such as `gfx1011` and `gfx1012`, see [Emulating
  as a Supported Architecture](#emulating-as-a-supported-architecture).
- `gfx1030`. This architecture can run programs compiled for `gfx103*` GPUs such as `gfx1031` and `gfx1032`, see [Emulating
  as a Supported Architecture](#emulating-as-a-supported-architecture).
- `gfx1010`
- `gfx1011`
- `gfx1012`

If your GPU is not on the list, please open an issue in our [issue tracker](https://github.com/getsolus/packages/issues/new/choose) with your GPU model and release year.

This list is only the minimum supported architectures. Some packages like [Blender](#blender) support more architectures.
```